packages/zudoku/src/config/validators/SidebarSchema.ts

```diff
@@ -32,6 +32,7 @@ export type SidebarItemCategory = Omit<
   items: SidebarItem[];
   link?: SidebarItemCategoryLinkDoc;
   icon?: LucideIcon | string;
+  apiReference?: boolean;
```

Comment:
Im not sure about the naming of this toggle. I think we should better describe the behaviour that we are toggleing here, like `openOnScroll`?

Reply:
- the key apiReference is not actually user-facing, this one is just to identify if the collapsable categories are apiRef or not, if they are not apis and just documentation I figured we should not add the open on scroll ability.

- the open-on-scroll actually defined by the key defaultCollapsed, that one is user-facing. ( I think here this should be defined as `openOnScroll` )

- Although, I am very much open to this suggestion.. what do you suggest ?